geonode/messaging/migrations/0001_initial.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.11 on 2020-03-27 14:40
+
+from django.db import migrations
+
+def create_notice_settings(apps, schema_editor):
+    """Its here only to trigger post_migration signal which will create messaging notification"""
+    pass
```

Comment:
Did you intend to make this pass? To me, it looks like this has no effect when you run migrations.

Reply:
I was tired of not fired post_migrate signal so i make empty migration which has no effect on the DB. But probably its not necessary

Reply:
If this sin't required please remove it. I don't like very much this kind of hacks @petrus7